src/pykoala/instruments/koala_ifu.py

```diff
@@ -858,163 +866,908 @@ def get_rss_mask(rss, **kwargs):   # THIS TASK SHOULD BE A CORRECTION CLASS, inc
+
+        telluric_correction = copy.deepcopy(_telluric_correction_)
+        
+        telluric_correction.telluric_correction = np.nanmedian(telluric_correction_list, axis=0)
+        
+        ptitle = "Telluric correction combining {} stars".format(n_stars)
+            
+        if telluric_correction_file is not None:
+            if path_to_data is not None:  telluric_correction_file = os.path.join(path_to_data,telluric_correction_file)
+            telluric_correction.save(filename = telluric_correction_file)
+    
+    else:  # no star_list provided but telluric_correction_file is, reading it    
+        if telluric_correction_file[-4:] == "fits":
+            # Read it from fits file #TODO
+            pass
+        else:            
+            if path_to_data is not None: telluric_correction_file = os.path.join(path_to_data, telluric_correction_file)
+            if verbose: print(" - Reading telluric correction from file",telluric_correction_file)
+            telluric_correction = TelluricCorrection(telluric_correction_file = telluric_correction_file)
+            
+        ptitle = "Telluric correction from file {}".format(telluric_correction_file)
 
+    if plot:
+        quick_plot(wavelength,telluric_correction.telluric_correction, 
+                  ptitle = ptitle,
+                  ylabel = "Telluric correction",
+                  **kwargs)
+        
+    return telluric_correction
 
+def quick_find_brightest_line(rss, 
+                              brightest_fibres_to_combine = None, 
+                              lowest_fibres_to_combine = None, 
+                              verbose= False):
+    """
+    Returns the wavelength where the peak of the brightest emission line is found.
 
+    Parameters
+    ----------
+    rss : object
+        rss 
+    brightest_fibres_to_combine : integer, optional
+        number of the brightest fibres to combine in the rss to do analysis. The default is 10.
+    lowest_fibres_to_combine : integer, optional
+        number of the lowest fibres to combine in the rss to do analysis. The default is 10.
+    verbose : boolean, optional
+        Print the result. The default is False.
 
+    Returns
+    -------
+    w_max : float
+        wavelegth where the peak of the brightest emission line is found.
+    """
+    if brightest_fibres_to_combine is None: brightest_fibres_to_combine = 10
+    if lowest_fibres_to_combine is None: lowest_fibres_to_combine = 10
+    fmax = np.nanmedian(rss.intensity[rss.koala.integrated_fibre_sorted[-brightest_fibres_to_combine:]], axis=0)
+    fmin = np.nanmedian(rss.intensity[rss.koala.integrated_fibre_sorted[:lowest_fibres_to_combine]], axis=0)    
+    s = fmax-fmin                                       
+    index_max = s.tolist().index(np.nanmax(s))
+    w_max = rss.wavelength[index_max] 
+    if verbose: print("> Quick checking wavelength of the brightest emission line in this rss:", w_max)
+    return (w_max)
```

Comment:
This is just the wavelength where a quick sky-subtracted spectrum reaches its maximum. It may well be the brightest emission line... or not.

What is it used for?

Reply:
Also, shouldn't it be in the `onedspec` module?